www/src/App.js

```diff
@@ -40,19 +21,15 @@ class App extends Component {
             <Link to="/">Home</Link>
           </Menu.Item>
           <Menu.Item>
-            <Link to="/merchant/login">Login</Link>
+            <Link to="/user/login">Login</Link>
           </Menu.Item>
           <Menu.Item>
-            <Link to="/merchant/signup">Signup</Link>
+            <Link to="/user/signup">Signup</Link>
           </Menu.Item>
         </Menu>
 
-        <Switch>
-          <Route exact path="/" component={Home} />
-          <Route path="/merchant/login" component={MerchantLogin} />
-          <Route path="/merchant/signup" component={MerchantSignup} />
-          <Route path="/merchant/:id" component={MerchantShop} />
-        </Switch>
+        <Route exact path="/" component={HomeApp} />
+        <Route path="/user" component={UserApp} />
```

Comment:
Removed the Switch because I figured its best to have all the base critical paths at this level then build out the user interfaces using React.